user_guide/first_steps.rst

```diff
@@ -0,0 +1,379 @@
+.. _user-guide-started:
+
+Ready, Set, Go
+==============
+
+OroCRM is a highly flexible system that can be tailored to meet the needs of your specific business.
+Major set-up is performed at the back-end during the system integration and described in more details in the
```

Comment:
a process or part of the program, not directly accessed by the user

shall I change the work to something else?